doc/api/net.md

```diff
@@ -40,8 +40,9 @@ added: v0.1.90
 
 * {Error}
 
-Emitted when an error occurs.  The [`'close'`][] event will be called directly
-following this event.  See example in discussion of `server.listen`.
+Emitted when an error occurs. Unlike the [`net.Socket`][], the [`'close'`][]
```

Comment:
Can you drop `the`, or replace it with `with`? (not a native speaker myself but the `the` sounds odd here)